### PR TITLE
Send Display Data to non-editable Weather components

### PR DESCRIPTION
**Pull request by ezequielc**
Along with https://github.com/Rise-Vision/common-template/pull/61, these changes improve Preview so that is sends Start event and Display data to the Template, in order for all weather components to receive display data.

I also reverted some of the changes from the previous strategy, as they are no longer relevant.

[stage-1]
To validate the changes, please proxy common-template.min.js requests to widgets.risevision.com/staging/common/2019.05.29.18.33/common-template.min.js

@alex-deaconu Please review. Thanks!

### CODE REVIEW

File: web/scripts/template-editor/directives/dtv-editor-preview-holder.js [hunk 1]
```diff
@@ -31,7 +29,9 @@ angular.module('risevision.template-editor.directives')
 
             _postAttributeData();
 
-            _setupComponents();
+            _postStartEvent();
```
Comment (alex-deaconu) [inline web/scripts/template-editor/directives/dtv-editor-preview-holder.js]: @ezequielc just to confirm, I don't believe this call is necessary as `_postAttributeData()` already calls the start event within RisePlayerConfiguration: https://github.com/Rise-Vision/common-template/blob/master/src/rise-attribute-data.js#L92. However, we can keep this here for good measure. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc looks good 👍 . Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc looks good 👍 . Thanks!

File: web/scripts/template-editor/directives/dtv-editor-preview-holder.js (hunk 1 above)
Comment (ezequielc) [reply web/scripts/template-editor/directives/dtv-editor-preview-holder.js]: @alex-deaconu Well noticed. I agree, let's keep it. Thanks

**Pull request merged**